sep490-frontend/src/app/modules/shared/services/toast-provider.ts

```
@@ -0,0 +1,52 @@
+  private readonly defaultLife = 1000;
+
+  constructor(private readonly messageService: MessageService) {
+    super();
+  }
+
+  businessError(message: ToastMessageOptions): void {
+    return this.messageService.add({
+      ...message,
+      severity: 'error',
+      life: this.defaultLife
+    });
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)